helm/cluster-aws/values.schema.json

```
@@ -472,6 +472,18 @@
             "type": "object",
             "title": "Control plane",
             "properties": {
+                "allowList": {
+                    "type": "array",
+                    "title": "Load balancer allow list",
+                    "description": "IPs that are allowed to connect to the control plane load balancer.",
```

Aligning this with other CIDR properties and their descriptions:

```suggestion
                    "description": "IPv4 address ranges that are allowed to connect to the control plane load balancer, in CIDR notation.",
```